addons/website_sale/views/templates.xml

```diff
@@ -25,9 +25,8 @@
     <template id="website_sale_layout" inherit_id="website.layout" priority="1">
         <body position="before">
             <t
-                t-if="website._get_product_image_ratio() != '1_1'"
                 t-set="body_classname"
-                t-value="'o_wsale_products_opt_thumb_' + website._get_product_image_ratio() + (' ' + body_classname if body_classname else '')"
+                t-value="'o_wsale_shop_ratio_' + website._get_product_image_ratio() + (' ' + body_classname if body_classname else '')"
```

Comment:
It's unlikely that we can merge this in prod.
I propose to keep the prev. nomenclature, but  adapt it as you did in the forward-port